src/AlertBox/AlertBoxIcons.less

```diff
@@ -0,0 +1,24 @@
jLJRjC8mMYwnIZhrC8rGXV1FNJ49qZWAZsQmBijh65zEXlaiq5VEK7aFRqQ54SbpVUFM+qf2WgXjzyhjmwFkiXyJpfMc6Vj0bl+NYVLW8aO1fAsepvH472OfFS1ouFPwX/1dZUJb1izcOTq/Abhp5sJ6o2qXh0TZfPVT26/l9UVFgL9BtIhVgoyrJscGcihI86nYZqoJVDzGzMPLTrdcuan8P9NzFCFlD9+DcUGgvcg05ZSVnt4KzV19uy3DuDcjgTLEkxN/P6VvgiI7PSfpFZyp6PfB5wBYxKZdhqA60VvNknMQ+Z3iTPBHFbUTZI2tjOBIkNHPOAefOdBCZh6qoN5E7hhg34BWFuwXknXKJ6oyyH7kXs8yik/Fun4kT2qGiMwLPZG2Gv70LKb3EMJDT5pX4MVBWhqRg1FdA0Um6oBl/G2bptQsYO9CMqdsOyrOLDxxb3lZJtGYR8pIjVo6Of1l6iTqrcfmYUl++dvgXBIDUxf3vfdHGQyrtayTJHbQNTtxqVU9eaQ+NVh+rmUfW94+wTOWuabronHnpf06rbwcVcLLD2bQ7SUiYX1PVhhQ2iy8WlUOplNEnvuAcYFhjQ71CKjf+r+th8nitVhdFxJN9O1LfR52AM/A/Yf0f1A9D3Y+hyDS7P95oTn2704WyZrqIX66foNzBrrblZugbc0HQD4iFHrY64yg18pwZxeqS5HOkh4GPdFeIBwCaAxeAT3bWM5lMAo/mMOT7A58xh0GQOgy3mMNhmzhrADnMY7DKHwR5zGHzBnHWAL5nDIGQOg4g5DJ4wJwB4yhwGXzGHwdfMYfANc+4DfMscBjFzGCTMYbCv6dYwzC1e0F2gtkFVoANTT1jcw+JQU2XI/o4Xhv29Qcz+wSCm/qjp9pD6Ey8M9WeDmPqLQUz9VdOdIfU3Xhjq7wYx9Q+DmPpMvxjLZQa/jHyXCgeUXWw+5++J9w/bxUC5AAEAAf//AA94nE1Sz28bRRR+b37vxl3b692xnRIn3XV2SxwZsrY3IRauVQnclKitKoNsqKgPRJQgYYTKBQQHVPUIF269VEj8AYVLiZC4VAip/AHc4MSPXriUCwo2s6aRGOl98/S+maf55ntA53/NP6WPaRM8iGAHnuunMTJe8wmhZADAYQqc8SkwoITRCRBEMgRCcARIcD+Oo7M7nuDLDY5RHAVCoijrsm6dSbbT7XTdk6KBQfQ8ttMuJmXtxw5mhU7c3n5SoT/PhksFt15Xs6GvlVt06bhUcJU/2xNMva1s2wATiEvyhsxZQsgPhRQWLZYs2y8e/xnUnZxl5ZzwNcW5rQ6VzZn1JleKzywqBJVgFp3/M79Hr9MlqEETurDXf/HsGhHcEUZKBYmRNwAFXCg+AWGkCTRaKQIjMAEmJRsCY3IEksl9X0dRVIzC0OIrjfVOO2pgMYNArKCnu+iHJ1mS0s7/mSTlnoNBk/QwqSHp2jK11VeZxE4nw7+lPbBPNNt48N73v//wjvjg28dHH18dm0ItY0zy9QJtPB4/Kdi/vP/g5s0HjzJY6AWYH9Jf6VUoQNU4G/frYVCllODAmAcjcyR3ASjNbMRTuBdvlEuMVxrYJIFDvBpJeqSNUafdw1ZSLpkPlyIMYtS+R797+tb9H+/funLlv+34t93rX0wOx58sb7V65wcH1/DRDjlz7fbly7e/eXiUbUcPZz+N35p8+Xp3dulcklRXdvGP0Y29k3feox8ZXxRU4GW41H/pIiq5epoYChG2isYfNgCp5BQUVVOBFOgUAM1gMsLeNVYRHJqjZMTNQJL9Zzb9+rmoU3Izc7bbTXSwhmmp6AkZBlFcbKflVqK5J7WvpTa6sgiDJsZRvOB7xEyugR5tJTUmysYw6RBpLn2mVzw08apXuJs3mXM3r8nnqxZWLcvSfK1+8cL6K8nGCyVDeqd3n4pc2xGciYKfr25UPGVmLadyqBi90+i/kV/0Qo2bszuLbnjgaPTyz4ZVdyNYDVf9XtxA18lXTrh+uOXagVfRlUDn3OXKmnvK39Qeyzmi/y+onpX6AAAAeJxjYGRgYADi9RGugvH8Nl8ZuJlfAEUYrhqICsLo/1//ZzF/YQ4CcjkYmECiACUsCv4AAHicY2BkYGAO+p8FJF/8//r/C/MXBqAICmAFALW2B48AA+gAAAPo//UDWf/9A+gAAAOgAAAAAAAAAHQA7AFAAeUAAQAAAAUAVgADAAAAAAACABIAIgBzAAAAXwtwAAAAAHicdZDdasIwGIbfzJ9tCtvYYKfL0VDG6g8MQRAEh55sJzI8HbXWtlIbSaPgbewedjG7iV3LXts4hrKWNM/35MuXrwFwjW8I5M8TR84CZ4xyPsEpepYL9M+Wi+QXyyVU8Wa5TP9uuYIHBJaruMEHK4jiOaMFPi0LXIlLyye4EHeWC/SPlovknuUSbsWr5TK9Z7mCiUgtV3EvvgZqtdVREBpZG9Rlu9nqyOlWKqoocWPprk2odCr7cq4S48excjy13PPYD9axq/fhfp74Oo1UIltOc69GfuJr1/izXfV0E7SNmcu5Vks5tBlypdXC94wTGrPqNhp/z8MACitsoRHxqkIYSNRo65zbaKKFDmnKDMnMPCtCAhcxjYs1d4TZSsq4zzFnlND6zIjJDjx+l0d+TAq4P2YVfbR6GE9IuzOizEv25bC7w6wRKcky3czOfntPseFpbVrDXbsuddaVxPCghuR97NYWNB69k92Koe2iwfef//sB5m6EUQB4nGNgYoAALgbsgJWRiZGZkYWRlZGNgb08sSgvMy+dJTMvLZ81tagov4i9uDQ5ObW4mIEBAKA1CgcAAAB4nGPw3sFwIihiIyNjX+QGxp0cDBwMyQUbGVidNjEwMmiBGJu5mBg5ICw+BjCLzWkX0wGgNCeQze60i8EBwmZmcNmowtgRGLHBoSNiI3OKy0Y1EG8XRwMDI4tDR3JIBEhJJBBs5mFi5NHawfi/dQNL70YmBhcADHYj9AAA') format('woff'),
```

Comment:
yeah, i'm probably in the minority here on this issue, but my thoughts are:
- svg fonts are better suited for icons that don't necessarily have a pre-determined size or color and can be adjusted easily to fit various contexts (grow with text, colored for different state like enabled/disabled, etc)
- plain svg icons are great for icons/images with pre-determined size/color settings (like our `Icon` component) but can be more restrictive WRT customization/adapting to surrounding context, since they have to be styled differently

in this specific case, no strong feelings either way, but figured this probably won't be the last time we need these icons in Dewey (e.g. toast notifications) and would be nice if they were flexible enough to be used in different situations